src/sidebar/services/test/load-annotations-test.js

```diff
@@ -326,6 +329,28 @@ describe('loadAnnotationsService', () => {
 
       assert.calledWith(onError, error);
     });
+
+    it('configures the streamer filter for uris', () => {
```

Comment:
I think this test description and the one following could be a little more clear. I think what we want to test here is that

* The streamer filter is set to filter on `uri` by (default OR `streamFilterBy` === `uri`) and `uris` are available
* The streamer filter is set to filter on `groups` if `streamFilterBy` === `groups` and `groupId` is available
* ...and what happens if `uris` is not available in the default case?
* ...and what happens if `streamFilterBy` === 'groups` but `groupId` is not available

Reply:
I added a bit more detail and I included the scenario were `uris` is either `[]` or is `undefined`.

`groupId` is mandatory so I didn't test the scenario when is not available.